symExec.py

```diff
@@ -734,8 +734,9 @@ def sym_exec_ins(start, instr, stack, mem, global_state, path_conditions_and_var
             raise ValueError('STACK underflow')
     elif instr_parts[0] == "MOD":
         if len(stack) > 1:
-            first = stack.pop(0)
-            second = stack.pop(0)
+            first = unsigned_format(stack.pop(0))
```

Comment:
This just seems to be a user defined function and all the operations used seem to be supported by z3 too